docs/release_notes.rst

```diff
@@ -36,6 +36,10 @@ VCE RARE
 Quality of Life Improvements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* We've added a new sub-command to ``dbt_helper`` - ``dbt_helper validate``.
+  This lets you run validation tests for a selection of DBT models and also
```

Comment:
It looks like "models" and "nodes" are used kinda interchangeably in this PR. Do we need to worry about situations where those two concepts are distinct, or does building a mental model where they are similar enough to stand in for one another get us valuable flexibility?

Reply:
Ah. Yes. We should say "node" because Sources are not Models but both are Nodes. Good catch.